minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/src/main/java/org/apache/nifi/minifi/bootstrap/BootstrapListener.java

```diff
@@ -257,7 +257,7 @@ private void handleBootstrapRequest(Socket socket) {
                 try {
                     socket.close();
                 } catch (IOException ioe) {
-                    logger.warn("Failed to close socket to Bootstrap due to {}", ioe.toString());
+                    logger.warn("Failed to close socket to Bootstrap", ioe);
```

Comment:
This is a slight behavior change, but I believe it is reasonable since closing the socket and failing can have different causes that would be useful to know.